compiler/tflchef/proto/tflchef.proto

```diff
@@ -274,6 +274,11 @@ message CosOptions {
   // None
 }
 
+message CumSumOptions {
+  optional bool exclusive = 1;
+  optional bool reverse = 2;
```

Comment:
The CumSum `reverse` option seems to have a default value - false.
How about adding it? =)

```suggestion
  optional bool exclusive = 1;
  optional bool reverse = 2 [default = false];
```

Note: https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/cumsum

Reply:
I also think adding default looks good :)

Reply:
Thanks, I'll apply it. 👍

Reply:
FYI, the default value of boolean type in protocol buffer v2 is `false`.
https://protobuf.dev/programming-guides/proto2/#optional:~:text=For%20bools%2C%20the%20default%20value%20is%20false.
Even though, the explicit assignment of default value may be helpful for preventing human errors.

In addition to, Protocol buffer v3 doesn't allow non-zero default value 🤔.